File: Mensuration/Triangle and Prism/Area/AoT.py
a=int(input("Enter 1st Side : "))
b=int(input("Enter 2nd Side : "))
c=int(input("Enter 3rd Side : "))
s=a+b+c/2
at=(s*(s-a)*(s-b)*(s-c))**0.5
print("Area : ",at)

File: Mensuration/Triangle and Prism/Area/AoT.cpp
#include<iostream>
#include<math.h>
using naamespace std;
void main(){
    int a,b,c;
    double s,t,area;
    cout<<"Enter 1st Side : ";
    cin>>a;
    cout<<"Enter 2nd Side : ";
    cin>>b;
    cout<<"Enter 3rd Side : ";
    cin>>c;
    s=(a+b+c)/2;
    t=s*(s-a)*(s-b)*(s-c);
    area=sqrt(t);
    cout<<"Area : "<<area;
}